posts/2022-03-18-BillyDawson_MeetTheTeam.adoc

```diff
@@ -0,0 +1,167 @@
+
+As part of the Docker squad I developed an understanding of how to use Docker containers, which was later complemented with Kubernetes to manage our containerised workloads and Helm to simplify and template containerised application deployment. I extended a simple team dashboard written with Ruby and CoffeeScript to include a tool to track customer cases, worked on several prototype projects – one of which became Microservice Builder - and helped automate some of our work using Jenkins and Travis CI.
+
+As part of the Wasdev team – the focus of which was mostly external and community based - I fixed and improved the performance of a webpage used for quickly identifying Liberty repositories, updated the web site and posted the monthly Liberty release updates for beta and GA releases and built an app to analyse traffic to the main Wasdev site. I also learnt how to use Maven and used this knowledge to modernise and add Maven support to many Liberty sample apps – this gave me exposure to a vast array of Java tools and APIs such as Derby & MongoDB, JDBC, JAXWS and JPA. Furthermore, I also had the pleasure of tinkering with an Arduino Uno and I extended our Docker dashboard to include a myriad of useful widgets – including our Git repositories, Travis CI status, security vulnerabilities, web page views and any relevant posts concerning Liberty on Twitter and Stack Overflow.
+
+Moreover, my placement year allowed me to enjoy a multitude of “give back” activities in addition to my day-to-day responsibilities. I attended a cross-site hackathon in Portsmouth and the University of Southampton’s Big Bang Fair in Southampton. I demonstrated the Liberty Car - an RC car with the radio controller replaced with a Raspberry Pi and using IBM Cloud to allow anyone at the conference to control the car via their mobile phone - at a WebSphere User Group and the Devoxx conferences in London. At our Hursley site I also hosted two primary schools at the XploreIT science and technology event, helped out at several Assessment Centres and attended coding dojos.
+
+Such an array of experiences helped tremendously with my last year of university – I learnt so many new technologies and deepened my knowledge of key areas featured within my course. This gave me a brilliant foundation for my final year and having experienced the application of technology from the perspective of enterprise I better understood the application of what I was studying rather than seeing it primarily from a theoretical perspective.
+
+Perhaps even more impactful were the soft skills I developed – my placement year significantly improved my overall confidence and communication skills. Confidence is something that I have personally struggled with and so the benefits of this cannot be overstated.
```

Comment:
suggested edit:

Perhaps even more impactful were the soft skills I developed. My placement year significantly improved my overall confidence and communication skills.

Reply:
this one